doc/setup/install.rst

```diff
@@ -475,6 +475,14 @@ Enterprise Edition: (see :ref:`setup/install/editions` to get access)
 
   $ git clone https://github.com/odoo/enterprise.git
 
+.. note:: Those repositories are quite heavy so you might be interrested only fetching the branch
```

Comment:
inte **r** esting :eyes:

Reply:
too late

Reply:
Just saying :D.
Thanks for the doc improvement :).